scripts/lima-k3s.sh

```diff
@@ -96,6 +104,14 @@ helm upgrade --install nginx ingress-nginx --repo https://kubernetes.github.io/i
 echo -e "${BLUE}\n:: Deploying base services to Kubernetes${NC}"
 kubectl apply -k "${REPO_DIR}/deploy"
 
+# Create Secret for ACR token
+echo -e "${BLUE}\n:: Creating Secret for ACR Token${NC}"
+kubectl delete secret ipaffs-acr || echo "Deleting secret ipaffs-acr"> /dev/null 2>&1
```

Comment:
This could be a little neater with:

```suggestion
kubectl delete secret ipaffs-acr 2>/dev/null || true
```